src/applications/disability-benefits/all-claims/constants.js

```diff
@@ -250,3 +250,12 @@ export const NULL_CONDITION_STRING = 'Unknown Condition';
 
 // Moment date format
 export const DATE_FORMAT = 'LL';
+
+export const WIZARD_STATUS = 'wizardStatus';
```

Comment:
NIT: A comment might be useful above this constant. It wasn't until I saw that it was being used as a key in `sessionsStorage` that I understood its purpose.